problemsets-mat223/modules/module4-exercises.tex

```diff
@@ -1,17 +1,115 @@
 \begin{exercises}
+	% Topics:
+	% dot products, orthogonality
+	% unit vectors, vector length,
+	% angle between vectors, distance between vectors
+	% lines, planes, hyperplanes in normal form
 	\begin{problist}
-		\prob  Example problem with parts
-		\begin{enumerate}
-			\item   The first part
-			\item   The second part
-		\end{enumerate}
-		\begin{solution}
-		Solutions to the parts
-			\begin{enumerate}
-				\item   $(2,4,8)$
-				\item   $(15,6,-21)$
-			\end{enumerate}
-		\end{solution}
+		\prob  Compute the following dot products.
+		\begin{enumerate}
+			\item   $\mat{9\\4} \cdot \mat{10\\-3}$
+			\item   $\mat{1\\36\\2} \cdot \mat{0\\0\\1}$
+			\item   $\mat{7\\6\\-3} \cdot \left(\mat{5\\11\\-1} + \mat{-2\\-6\\-1}\right)$
+			\item   $\mat{1\\3\\0\\-5\\5} \cdot \mat{1\\2\\2\\1\\2}$
+			\item   $\frac{1}{2}\mat{2\\5\\4} \cdot \mat{1\\0\\-1}$
+		\end{enumerate}
+		\prob Compute the length of the following vectors.
+		\begin{enumerate}
+			\item $\mat{2\\0}$
+			\item $\mat{1\\2\\3}$
+			\item $4\mat{5\\-6\\15\\2}$
+			\item $\mat{-\frac{\sqrt{10}}{10}\\-\frac{9\sqrt{10}}{50}\\\frac{12\sqrt{10}}{50}}$
+			\item $\mat{\frac{1}{3}\\\frac{\sqrt{3}}{3}\\0\\-\frac{1}{3}\\\frac{2}{3}}$
+		\end{enumerate}
+		\prob For each question, find the cosine of the angle between the pair of vectors.
```

Comment:
There are no questions....(do you see question marks?)

How about "For each pair of vectors listed below, ..."